### PR TITLE
Add support for Pelican 4

### PR DESCRIPTION
**Pull request by Ecno92**
Added support for Pelican 4. (before it raised a critical warning due to a key error)
Tests are working for both Pelican 3.7.1 as 4.0.1
https://travis-ci.org/Ecno92/image_process/builds/473609327

### CODE REVIEW

File: test_image_process.py [hunk 1]
```diff
@@ -285,17 +304,17 @@ def test_picture_generation(self, process_image):
                                 IMAGE_PROCESS_DIR='derivs')
         test_data = [
             ('<picture><source class="source-1" '
-             'src="/images/pelican-closeup.jpg"></source><img '
+             'src="/images/pelican-closeup.jpg"><img '
```
[the remark this answers is not in the document]
Comment (Ecno92) [reply test_image_process.py]: Thanks for your quick feedback. I've pushed a commit to fix it.

**Comment by Ecno92** [comment]
@patrickfournier could you maybe review again? Thanks!

**Comment by MinchinWeb** [comment]
@Ecno92 : I've been maintain [my own fork](https://github.com/MinchinWeb/minchin.pelican.plugins.image_process), and ran into this same issue. Thanks for providing the solution!

**Comment by Ecno92** [comment]
@MinchinWeb Thanks for your response and your efforts to keep maintaining this project using your own fork.

@patrickfournier What are your expectations regarding merging in open pull requests/maintaining this project?

**Comment by patrickfournier** [comment]
@Ecno92 or anyone else: would you like to be added as a maintainer? I appreciate the efforts you all put on creating PRs, but I feel someone else would do a better job at keeping this project up to date.

**Comment by Ecno92** [comment]
@patrickfournier Feel free to add me as an maintainer. For the short term I plan on working a little bit on my own website. So I may be able to help you as I probably will touch this extension as well.
For the long term I expect that I can not contribute to this project. However maybe someone else will show up to help. :-)

**Comment by patrickfournier** [comment]
@Ecno92 Thanks for volunteering. I added you as a maintainer:  https://github.com/whiskyechobravo/image_process/invitations

**Comment by Ecno92** [comment]
Most recent master includes the fixes for supporting pelican 4

**Pull request closed without merging**